src/tests/ftest/container/snapshot_info_test.py

```diff
@@ -0,0 +1,522 @@
+        except DaosApiError as error:
+            self.fail(
+                "##(1)Test failed during the initial object write: %s"
+                , str(error))
+        obj.close()
+        ##Take a snapshot of the container
+        snapshot = self.take_snapshot(self.container, epoch)
+        self.log.info("==(1)Container epoch= %s", epoch)
+        self.log.info("     snapshot.epoch= %s", snapshot.epoch)
+
+        #(2)Verify the snapshot is working properly.
+        try:
+            obj.open()
+            snap_handle = snapshot.open(
+                self.container.coh, snapshot.epoch)
+            thedata2 = self.container.read_an_obj(
+                len(thedata)+1, dkey, akey, obj, snap_handle.value)
+        except Exception as error:
+            self.fail(
+                "##(2)Error when retrieving the snapshot data: %s"
+                , str(error))
+        self.log.info("==(2)snapshot_list[ind]=%s", snapshot)
+        self.log.info("==snapshot.epoch=  %s", snapshot.epoch)
+        self.log.info("==written thedata=%s", thedata)
+        self.log.info("==thedata2.value= %s", thedata2.value)
+        if thedata2.value != thedata:
+            raise Exception("##(2)The data in the snapshot is not the "
+                            "same as the original data")
+        self.log.info("==Snapshot data matches the data originally "
+                      "written.")
+
+        #(3)Test snapshot with an invalid container handler
+        self.log.info("==(3)Snapshot with an invalid container handler.")
+        if self.invalid_snapshot_test(self.container, epoch):
+            self.log.info("==>Negative test 1, expecting failed on taking "
+                          "snapshot with an invalid container.coh: %s"
+                          , self.container)
+        else:
+            self.fail(
+                "##(3)Negative test 1 passing, expecting failed on"
+                " taking snapshot with an invalid container.coh: %s"
+                , self.container)
+
+        #(4)Test snapshot with a NULL container handler
+        self.log.info("==(4)Snapshot with a NULL container handler.")
+        if self.invalid_snapshot_test(None, epoch):
+            self.log.info("==>Negative test 2, expecting failed on taking "
+                          "snapshot on a NULL container.coh.")
+        else:
+            self.fail("##(4)Negative test 2 passing, expecting failed on "
+                      "taking snapshot with a NULL container.coh.")
+
+        #(5)Test snapshot with an invalid epoch
+        self.log.info("==(5)Snapshot with a NULL epoch.")
+        if self.invalid_snapshot_test(self.container.coh, None):
+            self.log.info("==>Negative test 3, expecting failed on taking "
+                          "snapshot with a NULL epoch.")
+        else:
+            self.fail("##(5)Negative test 3 passing, expecting failed on "
+                      "taking snapshot with a NULL epoch.")
+
+
+    def test_snapshot_info(self):
```

Comment:
Seems like this test is doing more than just testing snapshot information functionality. Hence the test name can be a bit misleading. We can change it to something more meaningful here. Like - snapshot_functionality or something. Also same goes for the name of the test and yaml files. The tests in side these files are doing more than just "info" test. So having something more meaning will be nice.
Can be "snapshot_functionality_test.py" If this covers all the important functional tests for snapshot. If you think some better please go ahead with that.